movie-reviews/src/app/components/core/RadioGroupComponent/radio-styles.css

```diff
@@ -0,0 +1,10 @@
+.chakra-radio__control {
+  display: none !important;
+}
+
+.chakra-radio:hover .rating-star,
+.chakra-radio:hover ~ .chakra-radio .rating-star,
```

Comment:
This could and should be done inside the component. You can directly add conditional coloring to the StarIcon like:
`color: {value > index ? 'yellow' : 'gray` or something like that.

Reply:
Could this or any other approach (sx maybe) handle all of these selectors?